### ISSUE TITLE
share button in loop grid

### ISSUE THREAD

**Issue by sach27298** [elementor/elementor#21617]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

when i use elementor share button in loop grid for post list for whatsapp button it shares blank title
i set dynamic post url in elementor target url. for single post i solve this issue with set target url as current page but when i do that in loop grid it shares that particular page's url(in which loop grid set for post listing) but i want to share that particular post. any ideas? even elementor code file or custom function to modify will great.

i set elementor share button in loop grid. its loop of post list and share button use for share that particular post. it share link prefect but issue is it also share blank title so it show ** then url

i want either to remove title or add proper title

### Steps to reproduce

i set elementor share button in loop grid. its loop of post list and share button use for share that particular post. it share link prefect but issue is it also share blank title so it show ** then url

### Isolating the problem

- [ ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ ] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [ ] I can reproduce this bug consistently following the steps above.

### System Info

					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: Percona Server (GPL), Release 23, Revision 500fcf5 v5.7.23-23
	PHP Version: 7.3.33
	PHP Memory Limit: 256M
	PHP Max Input Vars: 1000
	PHP Max Post Size: 516M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.1.1
	Site URL: https://arvallisamachar.com
	Home URL: https://arvallisamachar.com
	WP Multisite: No
	Max Upload Size: 512 MB
	Memory limit: 40M
	Max Memory limit: 256M
	Permalink Structure: /%postname%/
	Language: en-US
	Timezone: Asia/Kolkata
	Admin Email: support@nextway.co.in
	Debug Mode: Inactive

== Theme ==
	Name: Aravalli Samachar
	Version: 
	Author: nextway
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/110.0.0.0 Safari/537.36

== Active Plugins ==
	Advanced Access Manager
		Version: 6.9.7
		Author: Vasyl Martyniuk <vasyl@vasyltech.com>

	Blue Admin
		Version: 19.07.26
		Author: Linesh Jose

	Classic Editor
		Version: 1.6.2
		Author: WordPress Contributors

	Contact Form 7
		Version: 5.7.4
		Author: Takayuki Miyoshi

	Custom Dashboard & Login Page - AGCA
		Version: 7.1.4
		Author: Cusmin

	Disable WordPress Update Notifications
		Version: 2.3.3
		Author: Prem Tiwari

	Elementor
		Version: 3.11.2
		Author: Elementor.com

	Elementor Pro
		Version: 3.9.2
		Author: Elementor.com

	Integrate Google Drive (PRO)
		Version: 1.1.81
		Author: SoftLab

	Kirki Customizer Framework
		Version: 4.1.1
		Author: David Vongries

	Lazy Load for Comments
		Version: 1.0.10
		Author: Joel James

	Live News
		Version: 2.10
		Author: DAEXT

	One Click Demo Import
		Version: 3.1.2
		Author: OCDI

	Post Views Counter
		Version: 1.3.12
		Author: Digital Factory

	Powerkit
		Version: 2.8.4
		Author: Code Supply Co.

	Really Simple SSL
		Version: 6.2.0
		Author: Really Simple Plugins

	Regenerate Thumbnails
		Version: 3.1.5
		Author: Alex Mills (Viper007Bond)

	SearchWP Live Ajax Search
		Version: 1.7.2
		Author: SearchWP, LLC

	Secondary Title
		Version: 2.0.9.1
		Author: Kolja Nolte

	Site Kit by Google
		Version: 1.95.0
		Author: Google

== Elementor Experiments ==
	Optimized DOM Output: Active by default
	Improved Asset Loading: Active by default
	Improved CSS Loading: Active by default
	Inline Font Icons: Inactive by default
	Accessibility Improvements: Active by default
	Additional Custom Breakpoints: Active by default
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Inactive by default
	Upgrade Swiper Library: Active by default
	Default to New Theme Builder: Active by default
	Editor V2: Inactive by default
	Landing Pages: Active by default
	Nested Elements: Inactive by default
	Lazy Load Background Images: Inactive by default
	Save as Default: Active by default
	Page Transitions: Active by default
	Notes: Active by default
	Loop: Active by default
	Form Submissions: Active by default
	Scroll Snap: Active by default

== Log ==
	
JS: showing 7 of 7JS: 2023-02-27 10:26:28 [error X 2][https://arvallisamachar.com/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.11.2:3:791161] Cannot read properties of undefined (reading 'toLowerCase') 
JS: 2023-02-27 11:06:04 [error X 3][https://arvallisamachar.com/wp-content/plugins/elementor/assets/js/new-template.min.js?ver=3.11.2:2:2932] elementor_new_template_form_controls is not defined 
JS: 2023-02-27 11:13:11 [error X 1][https://arvallisamachar.com/wp-content/plugins/royal-elementor-addons/assets/js/lib/slick/slick.min.js?ver=1.8.1:1:19201] Cannot read properties of null (reading 'add') 
JS: 2023-02-28 03:22:11 [error X 11][https://arvallisamachar.com/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.11.2:3:1087291] Cannot read properties of null (reading 'getBoundingClientRect') 
JS: 2023-02-28 05:11:22 [error X 1][https://arvallisamachar.com/wp-content/plugins/elementor/assets/lib/pickr/pickr.min.js?ver=1.5.0:2:20189] Cannot read properties of null (reading 'inline') 
JS: 2023-02-28 06:09:01 [error X 2][https://arvallisamachar.com/wp-admin/load-scripts.php?c=1&loadchunk_0=underscore,jquery-core,jquery-migrate,backbone,jquery-ui-core,jquery-ui-mouse,wp-api-request,regenerator-runtime,wp-polyfill,wp-&loadchunk_1=hooks&ver=6.1.1:4:51824] Cannot read properties of undefined (reading 'replace') 
JS: 2023-03-10 03:32:30 [error X 56][https://arvallisamachar.com/wp-content/plugins/integrate-google-drive-premium/assets/js/elementor.js?ver=1.1.81:31:368280] elementor is not defined 

PHP: showing 3 of 3PHP: 2023-03-01 17:05:32 [notice X 3][/home3/arvalhuc/public_html/wp-content/plugins/elementor/core/common/modules/ajax/module.php::175] Undefined index: data [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-03-01 17:30:08 [notice X 2][/home3/arvalhuc/public_html/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php::167] Undefined index: editor_post_id [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-03-02 15:06:40 [notice X 712][/home3/arvalhuc/public_html/wp-content/plugins/elementor-pro/modules/posts/skins/skin-content-base.php::264] Trying to get property 'ID' of non-object [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatibility not specified

== Elementor Pro - Compatibility Tag ==

**Comment by Mesmer99** [comment]
Yup had same issue with blank url. I guess this is good functioanl.
